data/summary/store/store.go

```diff
@@ -15,6 +18,17 @@ import (
 	storeStructuredMongo "github.com/tidepool-org/platform/store/structured/mongo"
 )
 
+var (
+	QueueLag = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
You ought to pass buckets here, because the defaults are not suitable. From the docs:
```
var DefBuckets = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10}
```

> DefBuckets are the default Histogram buckets. The default buckets are tailored to broadly measure the response time (in seconds) of a network service. Most likely, however, you will be required to define buckets customized to your use case.

Perhaps this is a good starting point:
```
[]float64{0.5, 1, 2.5,  5, 7.5, 10, 25, 50, 75, 100, 150, 250, 500, 1000}
```